docs/fastdds/dds_layer/topic/contentFilteredTopic/createContentFilteredTopic.rst

```diff
@@ -30,21 +30,21 @@ Optional arguments are:
  * A string with the name of the filter class to use for the filter creation.
    This allows the user to create filters different from the standard SQL like one
    (please refer to :ref:`dds_layer_topic_contentFilteredTopic_custom_filters`).
-   Defaults to ``FASTDDS_SQLFILTER_NAME``.
+   Defaults to |FASTDDS_SQLFILTER_NAME-api|.
```

Comment:
API reference is not showing the actual value of the constant here

Reply:
Doxygen does not save the define value into the generated XML files. We can either add the value to the Doxygen documentation of the define or just write the value here (I have followed this last approach for the moment).

Reply:
That's good enough for now. I guess we'll need to make something to make the API reference show the values elsewhere